app/graphql/schema/type-definitions.js

```diff
@@ -162,8 +162,23 @@ type CustomerQueriesByTicketId {
   customerQueryResponses: [CustomerQueryResponse]
 }
 
+type OriginalCustomerQueryTicket {
```

Comment:
Wanted to highlight the need for this new type. When I first created the `customerQueriesBySbi` query, it would only return the original theory in the thread because it's only the top level query that actually holds additional values like `crn` and `sbi`. We originally decided this to avoid duplication of the same data throughout the entire thread of queries/responses. However, this was causing duplication because the top level query was being returned and then the thread array was returning the top level query again.

So to avoid this duplication I made a new `type def` seen above called `OriginalCustomerQueryTicket` which contains information only for the original query that was made. Reasoning behind this is that when an external user accesses the UI for customer queries, they'll see a list of all the original queries they made in descending order of the timestamp (i.e. most recent one at the top). 

What we could then do on the frontend is have a flow where once the external user clicks on a query, this will trigger another query (`customerQueriesByTicketId`) so that they can then view the full thread that came out of their original query.